sdks/python/build.gradle

```diff
@@ -226,6 +228,26 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
   }
 }
 
+task precommitIT(dependsOn: ['installGcpTest', 'sdist']) {
+  doLast {
+    // List of integration tests running in Python PreCommit.
+    def precommitTests = [
+        "apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it",
+        "apache_beam.examples.streaming_wordcount_it_test:StreamingWordCountIT.test_streaming_wordcount_it",
+    ]
+    def testOpts = [
```

Comment:
No need for `--attr=IT`, `--nologcapture`, `--nocapture`?

Reply:
done